samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
+@property (nonatomic) UISlider* fontSize;
```

Comment:
yep. re-grouped.

---
In reply to: [104796907](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r104796907) [](ancestors = 104796907)